packages/sel/sel.0.1.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis: "Simple Event Library"
+description: "Some sugar atop Unix.select"
+maintainer: ["Enrico Tassi"]
+authors: ["Enrico Tassi"]
+license: "MIT"
+tags: ["event" "input" "output"]
+homepage: "https://github.com/gares/sel"
+bug-reports: "https://github.com/gares/sel/issues"
+depends: [
+  "ocaml"
```

Comment:
```suggestion
  "ocaml" {>= "4.08.0"}
```

This is necessary due to dependency on various Stdlib modules only available in 4.08 or higher.